project-icp/bee-pollinator-app
# Data upload docs

## Overview

Add documentation for updating beekeeper rasters

Connects #386 

### Notes

The `beekeeper-data` credentials are in LastPass

## Testing Instructions

 * Confirm you can log in with the referenced IAM account, and that the account _only_ has access to the beekeepers raster s3 bucket, and no other services/buckets.
 * Confirm the steps listed in the docs are correct.
